src/components/TappableCopy.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {View, Pressable, Linking} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import compose from '../libs/compose';
+import {Checkmark, Clipboard as ClipboardIcon} from './Icon/Expensicons';
+import Clipboard from '../libs/Clipboard';
+import ReportActionContextMenuItem from '../pages/home/report/ReportActionContextMenuItem';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+
+const propTypes = {
+    /** Children to wrap in TappableCopy. */
+    children: PropTypes.node.isRequired,
+
+    /** Styles to be assigned to Container */
+    style: PropTypes.arrayOf(PropTypes.object),
+
+    /** Decides Tap behaviour. */
+    type: PropTypes.oneOf(['phone', 'email']),
+
+    /** Value to be copied or passed via tap. */
+    value: PropTypes.string.isRequired,
+
+    ...windowDimensionsPropTypes,
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    style: [],
+    type: undefined,
+};
+
+const TappableCopy = props => (
+    <View style={[styles.flexRow, styles.pRelative, ...props.style]}>
+        {props.type && props.isSmallScreenWidth
```

Comment:
Also can you split this file to index.js and index.native.js. Its a little weird changing the functionality based on screen width for this feature. Feels like it could easily break if we open this up to tablets with a bigger screen width so lets have index.js for web and index.native.js for mobile.

Reply:
Again, after changing the files we will have the same effect on M-web. So we have to use this logic.

Reply:
If we want to solve the issues for bigger tablet devices then we need to find a different approach.
Maybe something like this https://github.com/Expensify/Expensify.cash/issues/3296#issuecomment-875990808.

Reply:
Can you explain how the logic is the same? 
> Again, after changing the files we will have the same effect on M-web. So we have to use this logic.


For mobile I imagine that we just,
- show phone/email link

For web I imagine that we just,
- show text
- show copy to clipboard.

Then in this scenario we wouldn't need isSmallScreenWidth and withWindowDimensions right? Anyway let me know if I'm missing something.

Reply:
We also need to show phone/email link for M-web. so If I move this part to index.native.js. We still need the same conditions in index.js for the mobile web. Right?

Reply:
Ah m-web uses index.js. I totally missed that part and forgot about m-web. Thank you for explaining. You're good 👍

Reply:
Also if you didn't know (cause i didn't earlier today), marc informed me that we do have a helper method to detect mobile devices and that's [canUseTouchScreen](https://github.com/Expensify/Expensify.cash/blob/a26d1315cfbaeb717d0132316d42a11768a6ad7f/src/libs/canUseTouchscreen.js#L8-L32). I think your current code is good enough for this scenario but being aware of `canUseTouchScreen` would be cool for future solutions.